engines/scumm/players/player_mac_indy3.cpp

```diff
@@ -848,7 +848,6 @@ I3MPlayer::I3MPlayer(ScummEngine *vm, Audio::Mixer *mixer) : _vm(vm), _mixer(mix
 	assert(_musicChannels);
 	for (int i = 0; i < _numMusicChannels; ++i)
 		_musicChannels[i] = new MusicChannel(this);
-	memset(&_pcmSnd, 0, sizeof(_pcmSnd));
```

Comment:
Is this removal compensated somewhere else?

Reply:
It gets cleared in the constructor. I think I had the memset first, and later I added the constructor which makes the memset obsolete (and causes the non-trivial-object warning)

Reply:
Oh, I see. Thanks!